YouTrack/Connection.php

```diff
@@ -1645,8 +1645,10 @@ public function addValuesToEnumBundle($name, $values)
      */
     public function getProjectCustomField($project_id, $name)
     {
+        $xml = $this->get( '/admin/project/' . rawurlencode( $project_id ) . '/customfield/' . rawurlencode( $name ) );
```

Comment:
@Nitpick-CI can you tell me what coding standard I should apply?

It looks like I'm using a different one and knowing this would save everyone's time :)